docs/content/preview/yugabyte-platform/alerts-monitoring/universe-logging.md

```diff
@@ -30,7 +30,7 @@ If you want to set pgaudit.log_level to a [severity level](https://www.postgresq
 
 ## Limitations
 
-- Log export is only supported for cloud-based universes and automatically provisioned on-premises universes. Manually provisioned universes and Kubernetes universes are not supported.
```

Comment:
I don't get the context why we removed this?

Reply:
Support was added for manually provisioned universes. As a result, the only thing that is now not supported is K8s universes.